src/backend/base/langflow/api/v1/flows.py

```diff
@@ -32,13 +35,31 @@
 router = APIRouter(prefix="/flows", tags=["Flows"])
 
 
+async def _verify_fs_path(path: str):
+    if path:
+        path_ = Path(path)
+        if not await path_.exists():
+            await path_.touch()
```

Comment:
MAybe this could be in a `field_validator`?

Reply:
The problem is that `field_validator` is synchronous and here we do I/O ops.